ui/src/app/applications/components/application-deployment-history/application-deployment-history.tsx

```diff
@@ -42,6 +43,12 @@ export const ApplicationDeploymentHistory = ({
                             <br />
                             {(info.deployStartedAt && <Duration durationMs={moment(info.deployedAt).diff(moment(info.deployStartedAt)) / 1000} />) || 'Unknown'}
                         </div>
+                        <div>
```

Comment:
Although this is fine to me, maybe confirm with Leo if he wanted this to be on a separate line or on the same line as "Time to deploy" as depicted by his screenshot in the issue?